src/lib/ThumbnailsSidebar.js

```diff
@@ -239,6 +276,7 @@ class ThumbnailsSidebar {
         const thumbnailNav = document.createElement('button');
```

Comment:
Does a `div` with `role="button"` or `role="option"` serve us better here if we want to avoid allowing focus?

Reply:
Using a div with either of those roles still grabbed focus when clicked. The only ways I could prevent focus (pointer-events: none, or the disabled attribute) also prevented click events to the needed target.